central/compliance/manager/manager_impl.go

```diff
@@ -90,29 +96,41 @@ func newManager(standardsRegistry *standards.Registry, complianceOperatorManager
 }
 
 func (m *manager) createDomain(ctx context.Context, clusterID string) (framework.ComplianceDomain, error) {
+	t := time.Now()
 	cluster, ok, err := m.clusterStore.GetCluster(ctx, clusterID)
 	if err == nil && !ok {
 		err = errors.New("cluster not found")
 	}
 	if err != nil {
 		return nil, errors.Wrapf(err, "could not get cluster with ID %q", clusterID)
 	}
+	log.Infof("Took %d milliseconds to get cluster", time.Since(t).Milliseconds())
```

Comment:
Should these be metrics instead or are they debugging artifacts?

Reply:
Debugging, I removed them